examples/inline_osr.sou

```diff
@@ -0,0 +1,15 @@
+function main ()
```

Comment:
It would be nice to include an "after" version so that we don't have to run the program. Unless it's too large?